astroid/nodes/node_ng.py

```diff
@@ -446,15 +446,21 @@ def previous_sibling(self):
     # single node, and they rarely get looked at
 
     @cached_property
-    def fromlineno(self) -> int | None:
-        """The first line that this node appears on in the source code."""
+    def fromlineno(self) -> int:
+        """The first line that this node appears on in the source code.
```

Comment:
Maybe the docstring only need to be made explicit once in the parent class ?

Reply:
I don't know, I don't think all IDEs follow `docstrings` successfully but I don't really mind. I'll let you decide.

Reply:
The path of least resistance is merging as is, let's do that